lib/tasks/argo.rake

```diff
@@ -261,9 +261,12 @@ namespace :argo do
   end
 
   def load_order_files(fedora_files)
+    data_path = File.expand_path('../../../fedora_conf/data/', __FILE__)
```

Comment:
Seems like this might be part of another PR or a different commit. One looking at this commit down the road might wonder why this task was changed as part of this bug fix.